src/libs/snapshots.c

```diff
@@ -552,14 +560,15 @@ void gui_init(dt_lib_module_t *self)
     _clear_snapshot_entry(&d->snapshot[k]);
 
     /* create snapshot button */
-    d->snapshot[k].button = gtk_toggle_button_new_with_label(wdname);
-    GtkWidget *label = gtk_bin_get_child(GTK_BIN(d->snapshot[k].button));
-    gtk_widget_set_halign(label, GTK_ALIGN_START);
-    gtk_label_set_xalign(GTK_LABEL(label), 0);
-    gtk_label_set_ellipsize(GTK_LABEL(label), PANGO_ELLIPSIZE_MIDDLE);
+    d->snapshot[k].button = gtk_toggle_button_new();
```

Comment:
:-)

@tpinfold This is why we are always hesitant to add these corner features that only appeal to a few people; they always cause complications later (and even if this one is solvable, the next one will need to work with the workaround etc etc).

@TurboGit When creating a new snapshot we could always put a label in there (deleting whatever else, if anything, is currently there). And then when the user ctrl+clicks, create an entry, fill it with the current contents of the label, delete the label, put the entry in its place and focus it. Of course this would be one-way only; if you ctrl+click, you lose the <small> formatting. And you don't get the ** when you switch to another image.

I wouldn't want to go any further than that; no attempts to polish and make it work _just_ right; I just don't have enough duct tape for that. Not even publish this feature in a tooltip. And maybe not even promise that we won't yank it out when it gets too ugly...

Reply:
Ok, I'll merge this and I'll just add the multi-name if not empty without markup for now.